libs/ui/src/ui_strings/app_strings.tsx

```diff
@@ -57,49 +57,33 @@ export const appStrings = {
     // better voter experience.
     // This pattern only makes sense when there's a very limited value space.
     ({
-      1: (
-        <UiString uiStringKey="numSeatsInstructions" uiStringSubKey="1">
-          Vote for 1.
-        </UiString>
-      ),
-      2: (
-        <UiString uiStringKey="numSeatsInstructions" uiStringSubKey="2">
-          Vote for 2.
-        </UiString>
-      ),
-      3: (
-        <UiString uiStringKey="numSeatsInstructions" uiStringSubKey="3">
-          Vote for 3.
-        </UiString>
-      ),
-      4: (
-        <UiString uiStringKey="numSeatsInstructions" uiStringSubKey="4">
-          Vote for 4.
-        </UiString>
-      ),
+      1: <UiString uiStringKey="numSeatsInstructions.1">Vote for 1.</UiString>,
+      2: <UiString uiStringKey="numSeatsInstructions.2">Vote for 2.</UiString>,
+      3: <UiString uiStringKey="numSeatsInstructions.3">Vote for 3.</UiString>,
+      4: <UiString uiStringKey="numSeatsInstructions.4">Vote for 4.</UiString>,
```

Comment:
My reading of this is that a caller will get `undefined` for anything that isn't 1, 2, 3 or 4, so the "Vote for N." label will just be missing. Is that right? If so, shouldn't we have a fallback that does interpolation?

Reply:
Yeah, was considering that, but want to avoid interpolation altogether where possible -- was going to see if we can reliably assert an upper limit, otherwise fall back to a non-interpolated string like "Number of votes available: x"

Reply:
I'm sure there is an upper limit, but it might be an absurd number like 50. I've seen as high as 13 in my own experience as a voter.

Reply:
Oh thanks, that's a lot more than I thought! I think I'll just reword these strings, then, as long as they don't sound too clunky